ListingManager.Tests/ListingManager.Tests.csproj

```diff
@@ -1,7 +1,7 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFramework>netcoreapp2.2</TargetFramework>
+        <TargetFramework>netcoreapp3.1</TargetFramework>
 
```

Comment:
While you're at it, lets bump the LangVersion to 8. We can go ahead and do this on all the projects `<LangVersion>8.0</LangVersion>`

Reply:
`DirectoryBuild.props`?